templates/organization/base-organization.html

```diff
@@ -23,43 +23,23 @@ <h1>
-                <a href="{% url 'edit-organization-composition' organization.uuid current_composition.id %}" class="btn btn-primary">{% trans "Relationships" %}</a>
-            {% elif first_composition %}
-                <a href="{% url 'edit-organization-composition' organization.uuid first_composition.id %}" class="btn btn-default">{% trans "Relationships" %}</a>
-            {% elif organization %}
-                <a href="{% url 'create-organization-composition' organization.uuid %}" class="btn btn-default">{% trans "Relationships" %}</a>
-            {% endif %}
-            {% if first_personnel and current_membership %}
-                <a href="{% url 'edit-organization-personnel' organization.uuid first_personnel.id %}" class="btn btn-primary">{% trans "Personnel" %}</a>
-            {% elif first_personnel %}
-                <a href="{% url 'edit-organization-personnel' organization.uuid first_personnel.id %}" class="btn btn-default">{% trans "Personnel" %}</a>
-            {% elif organization %}
-                <a href="{% url 'create-organization-personnel' organization.uuid %}" class="btn btn-default">{% trans "Personnel" %}</a>
-            {% endif %}
-            {% if first_emplacement and current_emplacement or first_association and current_association %}
-                {% if current_emplacement %}
-                <a href="{% url 'edit-organization-emplacement' organization.uuid first_emplacement.id %}" class="btn btn-primary">{% trans "Locations" %}</a>
-                {% elif current_association %}
-                <a href="{% url 'edit-organization-association' organization.uuid first_association.id %}" class="btn btn-primary">{% trans "Locations" %}</a>
-                {% endif %}
-            {% elif first_emplacement or first_association %}
-                {% if first_emplacement %}
-                <a href="{% url 'edit-organization-emplacement' organization.uuid first_emplacement.id %}" class="btn btn-default">{% trans "Locations" %}</a>
-                {% else %}
-                <a href="{% url 'edit-organization-association' organization.uuid first_association.id %}" class="btn btn-default">{% trans "Locations" %}</a>
-                {% endif %}
-            {% elif organization %}
-                <a href="{% url 'create-organization-emplacement' organization.uuid %}" class="btn btn-default">{% trans "Add site" %}</a>
-                <a href="{% url 'create-organization-association' organization.uuid %}" class="btn btn-default">{% trans "Add area" %}</a>
+            {% if organization.uuid %}
+              <a href="{% url 'edit-organization' organization.uuid %}"
+                  class="btn btn-{% if is_basics_active %}primary{% else %}default{% endif %}">{% trans "Basics" %}</a>
+              <a href="{% url "create-organization-composition" organization.uuid %}"
+                  class="btn btn-{% if is_composition_active %}primary{% else %}default{% endif %}">{% trans "Relationships" %}</a>
```

Comment:
Since the `Relationships` tab encompasses both `Compositions` and `Memberships`, how about we change this active variable to `is_relationship_active`?